src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisCard.tsx

```diff
@@ -0,0 +1,724 @@
+              onClick={usingArchivedReport ? startProfilingRecording : clearCacheStartRecording}
+            />
+          </Tooltip>
+        </Text>
+      </TextContent>
+    );
+  }, [
+    isLoading,
+    usingArchivedReport,
+    usingCachedReport,
+    reportStalenessTimer,
+    reportStalenessTimerUnits,
+    clearCacheStartRecording,
+    startProfilingRecording,
+  ]);
+
+  const toolbar = React.useMemo(() => {
+    if (isError) {
+      return null;
+    }
+    return (
+      <Toolbar
+        id="automated-analysis-toolbar"
+        aria-label="automated-analysis-toolbar"
+        clearAllFilters={handleClearFilters}
+        clearFiltersButtonText="Clear"
+        isFullHeight
+      >
+        <ToolbarContent>
+          <AutomatedAnalysisFilters
+            target={targetConnectURL}
+            evaluations={categorizedEvaluation}
+            filters={targetAutomatedAnalysisFilters}
+            updateFilters={updateFilters}
+          />
+          <ToolbarGroup>
+            <ToolbarItem>
+              <Button
+                isSmall
+                isAriaDisabled={isLoading || usingCachedReport || usingArchivedReport}
+                aria-label="Refresh automated analysis"
+                onClick={generateReport}
+                variant="control"
+                icon={<Spinner2Icon />}
+              />
+              <Button
+                isSmall
+                isAriaDisabled={isLoading || usingArchivedReport}
+                aria-label="Delete automated analysis"
+                onClick={clearAnalysis}
+                variant="control"
+                icon={<TrashIcon />}
+              />
+            </ToolbarItem>
+            <ToolbarItem>
+              <Checkbox
+                label="Show N/A scores"
+                isChecked={showNAScores}
+                onChange={handleNAScoreChange}
+                id="show-na-scores"
+                name="show-na-scores"
+              />
+            </ToolbarItem>
+          </ToolbarGroup>
+        </ToolbarContent>
+      </Toolbar>
+    );
+  }, [
+    isLoading,
+    isError,
+    showNAScores,
+    targetConnectURL,
+    categorizedEvaluation,
+    targetAutomatedAnalysisFilters,
+    generateReport,
+    handleClearFilters,
+    handleNAScoreChange,
+    updateFilters,
+  ]);
+
+  const errorView = React.useMemo(() => {
+    return (
+      <ErrorView
+        title={'Automated Analysis Error'}
+        message={
+          <TextContent>
+            <Text component={TextVariants.p}>Cryostat was unable to generate an automated analysis report.</Text>
+            <Text component={TextVariants.small}>{errorMessage}</Text>
+          </TextContent>
+        }
+        retryButtonMessage={handleErrorView()[0]}
+        retry={handleErrorView()[1]}
+      />
+    );
+  }, [errorMessage, handleErrorView]);
+
+  const view = React.useMemo(() => {
+    if (isError) {
+      if (errorMessage == 'Authentication failure') {
```

Comment:
Need to use

https://github.com/cryostatio/cryostat-web/blob/54c6853e60ecd067f412a5fa83bc59df41fc6993/src/app/ErrorView/ErrorView.tsx#L51

For consistency and easy management?